tools/donate_cpu_lib.py

```diff
@@ -0,0 +1,478 @@
+# Donate CPU client library
```

Comment:
I suggest using `-` in the file name instead of `_` to be consistent with the other `donate-cpu*` scripts.

Reply:
AFAIU we can't import file with dashes in name by-design: https://bugs.python.org/issue909226

Reply:
Oh, ok. I didn't know that, I never used dashes in file names. Then we have to live with it.